src/components/TextFilter/TextFilter.css

```diff
@@ -0,0 +1,32 @@
+.TextFilter .text-input input {
```

Comment:
I know that this is not the same that we do in each dapp repo,  but on this lib we follow the semantic ui style for class names, and we add the class `dcl` to all the components added by this lib (while all the components that come from semantic-ui have a `ui` class) and uses dash-case for the class names.

Following that convention, this component should have the classes `.dcl.text-filter`

Reply:
That's perfect, changed it!